### ISSUE TITLE
Add high-resolution shapefile of Australia's coastline for plotQC's map

### ISSUE THREAD

**Issue by fmaron** [IMOS-AnimalTracking/remora#6]
**Is your feature request related to a problem? Please describe.**

Joni's comments: 

The plots are very cool, though I feel the resolution for the one on the right isn't high enough for small arrays or where receivers/detections are close to the coast. Many of our receivers are 50-300m from the coast and so most of our detections looked like they were on land! 

**Describe the solution you'd like**

Higher resolution map to avoid detections "on land".

**Describe alternatives you've considered**

@ianjonsen comments:
It will require more work to find a suitable coastline database for fine-scale projects. We probably need a high-res shapefile of Australia's coastline that wouldn't be available from any of R's mapping or spatial database pkg's. 

**Additional context**

**Comment by fjaine** [comment]
I agree with both Joni's and @ianjonsen comments - at this stage leaflet / openstreet maps are as good as it gets I believe.

Higher-resolution coastline data will come as heavy shapefiles for each state, which are both 1) hard to source for all states and 2) heavy to load via the package.

@vinayudyawer  might have some ideas or insights to share on this?

**Comment by fjaine** [comment]
This will also be an issue for users from other networks/countries... Not much we can do for those who work in estuarine/river settings. It might be difficult for us to source those high-res shapefiles for all countries and host them on the repo? Unless there is an online repository for them somewhere..?

**Comment by fjaine** [comment]
I would like to see this QC output map come as an interactive leaflet map, which users can explore by clicking on specific erroneous detections and visualising rows of data where the issue lies, etc.

**Comment by ianjonsen** [comment]
Will explore adapting QC map code to use {leaflet} for interactivity and high-res coastline data. For estuarine/river cases, or any other situation where a higher-res option would be desirable, we could provide option for user to upload their own shapefile.
